src/App.css

```diff
@@ -21,48 +40,183 @@ h1 {
 
 .link {
   text-decoration: none;
-  color: #000;
+  color: #121212;
+  font-family: 'Montserrat', serif;
   margin-left: 1rem;
   padding: 1rem;
 }
 
 .book-container {
   display: grid;
   grid-template-columns: 40% 30% 30%;
-  margin: 2.375rem 7.14% 0.938rem 7.14%;
+  margin: 0 7.14% 0.938rem 7.14%;
   background-color: #fff;
   border: 1px solid #e8e8e8;
-  height: 10.625rem;
+  height: 6.625rem;
+  padding: 2rem;
 }
 
-.book-info {
-  margin-left: 1.688rem;
-}
-
-.book-genre {
+.book-category {
   color: #121212;
   font-size: 0.875rem;
-  opacity: 50%;
+  opacity: 0.5;
+  margin: 0 0 0.25rem 0;
+  font-family: 'Montserrat', serif;
+  font-weight: bold;
 }
 
 .book-title {
   color: #121212;
   font-size: 1.375rem;
+  margin: 0;
+  font-family: 'Roboto Slab', sans-serif;
+  font-weight: bold;
 }
 
 .book-author {
   color: #4386bf;
   font-size: 0.875rem;
+  margin: 0;
+  font-family: 'Roboto Slab', sans-serif;
+  font-weight: 300;
 }
 
 .remove-btn {
-  margin-left: 2rem;
+  width: 3.375rem;
+  height: 1.188rem;
+  margin: 1.33rem 0.938rem 0 0;
+  font-family: 'Roboto Slab', sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  border: none;
+  background-color: #fff;
+  cursor: pointer;
+}
```

Comment:
Please ensure that your final design should match Zeplin's design.

Your Implementation | Zeplin's design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/161154792-b0a1904f-31fb-4b37-8fec-0380b143e012.png) | ![image](https://user-images.githubusercontent.com/10332499/161154707-a20a32ad-9a3a-4b34-ae01-152c3b937362.png)

Reply:
All good now 👍🏽